### ISSUE TITLE
Add Documentation on Extending RhumRunner

### ISSUE THREAD

**Issue by ebebbington** [drashland/rhum#9]
## Summary

What: Add documentation to the README on how to extend the `RhumRunner`:  why, how, use cases for doing do etc

Why: The runner is exposed for users to extend if they wish (for whatever purpose, thanks to @crookse), but this documentation is missing.

## Acceptance Criteria

- Update README to include full documentation about extension the runner:
    - Why
    - How
    - Use case for doing so

**Comment by ebebbington** [comment]
I can't think of a use case for extending it, though i'm not saying it isn't useful. Open to anyone else as I've spent enough time on it getting nowhere

I reckon it's still worth adding documentation that **its possible to extend** as the bare minimum

**Comment by crookse** [comment]
I agree. We should state that the rhumrunner is exposed. Sall good. We can put this on the back burner

**Comment by ebebbington** [comment]
If you or @Guergeiro can think of anything then of course have a play

**Comment by crookse** [comment]
moving this to the website repo
